src/Dialogs/Weather/RASPDialog.cpp

```diff
@@ -115,11 +116,16 @@ RASPSettingsPanel::OnTimeModified(const DataFieldEnum &df) noexcept
 void
 RASPSettingsPanel::Download() noexcept
 {
-  auto path = DownloadFilePicker(FileType::RASP);
-  if (path == nullptr)
+  auto downloaded_file_path = DownloadFilePicker(FileType::RASP);
+  if (downloaded_file_path == nullptr)
     return;
+  
+  auto rasp_file_path = AllocatedPath::Build(downloaded_file_path.GetParent(), RASP_FILENAME);
+  
+  // Replace the old RASP file with the downloaded one
```

Comment:
This code comment is useless. It only states what is obvious from reading the code, but it doesn't answer the "why" question.